src/WinEncoding.jl

```diff
@@ -113,7 +113,7 @@ module WinEncoding
     [0x8d] => '\\u8d' as windows api does;no invalid sequence error;non-blocking
     """
     function decode1252(ss::Vector{UInt8}) 
-        o=zeros(UInt8,length(ss)*3)
+        o=similar(ss, length(ss)*2+1)
```

Comment:
This will cause buffer overflow 
@test codeunits(decode1252([0x80,0x80,0x80]))==[0xe2, 0x82, 0xac, 0xe2, 0x82, 0xac, 0xe2, 0x82, 0xac]

Reply:
"Microsoft updated their version of code page 950 in 2000, adding the euro sign (€) at the double-byte code 0xA3E1. IBM refers to the euro sign update as CCSID 1370 (which includes both single-byte (0x80) and double-byte euro signs)"